pages/Reference/glossary.md

```diff
@@ -4,4 +4,24 @@ route: /Reference/Glossary
 menu: Reference
 ---
 
-TBD
+# Glossary
+
+* **Account** - See "Social Identity"
+* **Announcement** - A item of data, typically an image or note, either directly posted or hashed then posted to the blockchain
+* **Announcer** - The contract that posts new content from a publisher to the blockchain through batches or batch announcements by an archivist
+* **Publisher** - A user or bot who posts new content to an announcer
+* **Bad actor** - A user intentionally using the application for bad-faith or illegal purposes
+* **Bot** - An automated account, sometimes malicious but often providing some service ("Hey Alexa, what's the weather?")
+* **Consumer** - Someone who reads content from social media
+* **Contract address** - The unique number associated with a smart contract posted on the blockchain
+* **Happy path** - The expected user flow assuming no errors or bad actors
```

Comment:
Let's drop this one too. It's more of any industry term than a project-specific thing.